client-api/src/main/java/org/sonarsource/sonarlint/core/clientapi/client/connection/AssistCreatingConnectionParams.java

```diff
@@ -19,15 +19,32 @@
  */
 package org.sonarsource.sonarlint.core.clientapi.client.connection;
 
+import javax.annotation.Nullable;
+
 public class AssistCreatingConnectionParams {
   private final String serverUrl;
+  @Nullable
+  private final String tokenName;
```

Comment:
I've included the `tokenName` alongside the `tokenValue` if at some point we want to store this information on the IDE side. As an example: When the user uses the "Open in IDE" feature but cancels (I dunno why) and then proceeds to connect manually, we could already have the token in stock. Of course only for some time and not persistent ^^

Reply:
The `tokenName` is now also used when revoking tokens from the IDE side after a failed automatic connected mode setup.